.github/CODEOWNERS

```diff
@@ -74,3 +74,8 @@
 # ./lib/internal/modules/* @nodejs/modules
 # ./lib/internal/bootstrap/loaders.js @nodejs/modules
 # ./src/module_wrap* @nodejs/modules @nodejs/vm
+
+# N-API
+
+# ./src/node_api* @nodejs/n-api
+# ./src/js_native_api* @nodejs/n-api
```

Comment:
Should the relevant doc pages also require sign off?

Reply:
@richardlau makes sense added.